src/legacy/ui/public/visualize/loader/embedded_visualize_handler.ts

```diff
@@ -435,7 +436,13 @@ export class EmbeddedVisualizeHandler {
     this.fetchAndRender();
   };
 
+  private cancel = () => {
```

Comment:
Should this be `public`?

Reply:
nope, else it should probably part of embeddable API

Reply:
yup, i think so, but not necessary for now
we'll want to cancel externally (once we have cancel buttons around)